src/wp-includes/functions.php

```diff
@@ -4491,6 +4491,13 @@ function wp_send_json_error( $data = null, $status_code = null, $options = 0 ) {
 						'message' => $message,
 					);
 				}
+				if ( is_null( $status_code ) ) {
+					$error_data = $data->get_error_data( $code );
+					if ( ! empty( $error_data['status'] ) ) {
+						$status_code = $error_data['status'];
+						$result[]    = $error_data;
```

Comment:
```suggestion
						$result[]    = $error_data;
```

What is this line needed for?

Reply:
I added that so that if additional data is passed in the WP_Error object then that can be shown.
But it does not affect anything so it can be removed.
Let me know if I should remove this.
Thank you.

Reply:
@rohitmathur-7 Let's remove it for now.

Reply:
Removed the line.